lib/ansible/modules/cloud/amazon/iam_server_certificate_info.py

```diff
@@ -10,22 +10,28 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: iam_server_certificate_info
 short_description: Retrieve the information of a server certificate
 description:
-  - Retrieve the attributes of a server certificate.
+  - Retrieve the attributes of a server certificate
```

Comment:
```suggestion
  - Retrieve the attributes of a server certificate.
```

Description fields should be complete sentences.

Reply:
See also https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block